commit 7d4caa6a60af0e584dc5c3dc44437117744f6f84
Author: Jelmer VernooÄ³ <jelmer@jelmer.uk>
Date:   Sat Jan 14 20:55:00 2017 +0000

    Use sys/socket.h rather than asm/socket.h.
    
    The latter is not available on some platforms.

diff --git a/core/quasselc-connector.c b/core/quasselc-connector.c
index 08a3718..77e8d7a 100644
--- a/core/quasselc-connector.c
+++ b/core/quasselc-connector.c
@@ -16,7 +16,7 @@
  */
 
 #define _GNU_SOURCE
-#include <asm/socket.h>
+#include <sys/socket.h>
 #include <sys/types.h>
 #include <sys/socket.h>
 #include <sys/wait.h>
